docker-storage-setup.sh

```diff
@@ -422,9 +425,10 @@ reset_extra_volume () {
 }
 
 reset_lvm_thin_pool () {
+  local thinpool_name=${CONTAINER_THINPOOL:-docker-pool}
   if lvm_pool_exists; then
```

Comment:
--reset is broken. you need to pass in name of thin pool to lvm_pool_exists.

if lvm_pool_exists $thinpool_name; then
fi